src/wizards/clusters.ts

```diff
@@ -231,4 +205,35 @@ export async function configureDefaultClusters(clusterSettings: ClusterSettings)
     ];
 }
 
+export async function addClusters(clusters: Cluster[], clusterSettings: ClusterSettings, explorer: KafkaExplorer) {

```

Comment:
we should prolly move that out from the wizard namespace

Reply:
For me addCluster should be renamed to saveCluster and it's a command handler. I suggest that you create in commands/clusters.ts a new class 

```ts
export class SaveClusterCommandHandler {

    constructor(private clusterSettings: ClusterSettings, private explorer: KafkaExplorer) {
    }

    async execute(clusters: Cluster[]): Promise<void> {
        try {
            // Save collected clusters in settings.
            let createdClusterNames = '';
            for (const cluster of clusters) {
                this.clusterSettings.upsert(cluster);
                if (createdClusterNames !== '') {
                    createdClusterNames += '\', \'';
                }
                createdClusterNames += cluster.name;
            }
            vscode.window.showInformationMessage(`${clusters.length > 1 ? `${clusters.length} clusters` : 'Cluster'} '${createdClusterNames}' created successfully`);
    
            // Refresh the explorer
            this.explorer.refresh();
    
            // Selecting the created cluster is done with TreeView#reveal
            // 1. Show the treeview of the explorer (otherwise reveal will not work)
            this.explorer.show();
            // 2. the reveal() call must occur within a timeout(),
            // while waiting for a fix in https://github.com/microsoft/vscode/issues/114149
            setTimeout(() => {
                if (clusters) {
                    this.explorer.selectClusterByName(clusters[0].name);
                }
            }, 1000);
        }
        catch (error) {
            showErrorMessage(`Error while creating cluster`, error);
        }
    }
}
```

and you consume it in extension.ts and wizard both.